apps/system-e2e/src/tests/islandis/service-portal/smoke/company-delegation.spec.ts

```diff
@@ -43,7 +43,42 @@ test.describe('Service portal', () => {
       waitUntil: 'domcontentloaded',
     })
 
+    const dashboard = page.getByTestId('service-portal-dashboard')
+
     // Assert
     await expect(findByRole('heading', companyName ?? '')).toBeVisible()
+    await expect(dashboard).toBeVisible()
+    await expect(await dashboard.locator('a').count()).toBeLessThan(10)
+  })
+
+  test('can view company data', async () => {
+    // Arrange
+    const page = await context.newPage()
+    const { findByRole } = helpers(page)
+    await page.goto('/minarsidur?locale=is&hide_onboarding_modal=true')
+
+    // Act
+    await page.locator('data-testid=user-menu >> visible=true').click()
+    await page.locator('role=button[name="Skipta um notanda"]').click()
+    const firstCompany = page.locator('role=button[name*="Prókúra"]').first()
+    await expect(firstCompany).toBeVisible()
+    const companyName = await firstCompany
+      .locator('.identity-card--name')
+      .textContent()
+    expect(companyName).toBeTruthy()
+    await firstCompany.click()
+    await page.waitForURL(new RegExp(homeUrl), {
+      waitUntil: 'domcontentloaded',
+    })
+
+    const link = findByRole('link', 'Um fyrirtæki').first()
```

Comment:
FYI, the `findByRole` helper method is no longer necessary since `page` now has native `getByRole`.